user_guide/common_features_channels.rst

```diff
@@ -3,13 +3,45 @@
 Channels
 ========
 
-Collection of comprehensive customer-related data is a cornerstone of customer relations management.
 
-To represent different sources of customer-related data, OroCRM uses records of the Channel entity ("channels").
+Today, successful businesses usually have more than one sales site. These may be different online stores, 
+business-to-business enterprises, or business-to-customer outlets. Moreover, different customer and sales-related 
+information can be received from various survey-campaigns or membership in clubs, funds, charity events, etc. Gaining a 
+full understanding of all the information you receive from each of these sources, along with all the individual customer 
+information you collect from various other sources, becomes crucial. With OroCRM's Channels you can do both with minimum 
+effort. 
```

Comment:
Edit: Today, successful businesses usually have more than one sales site. These may be different online stores, business-to-business enterprises, or business-to-customer outlets. Moreover, different customers and sales-related data can be received from various survey-campaigns or memberships in clubs, funds, charity events, etc. Gaining a full understanding of all the information you receive from each of these sources, along with all the individual customer information you collect from various other sources, becomes crucial. With OroCRM's Channels, you can do both with minimum effort.